.gitignore

```diff
@@ -1,5 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+/.vscode
```

Comment:
This folder was probably already committed before you added this rule, right? You can run this command to remove it from git tracking. [See here](https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore) for more info

```
git rm -r --cached .vscode
```